Resources/Prototypes/Research/industrial.yml

```diff
@@ -145,10 +145,11 @@
     state: icon
   discipline: Industrial
   tier: 2
-  cost: 5000
+  cost: 7500
   recipeUnlocks:
-  - HolofanProjector
```

Comment:
put holofan in a lathe instead of removing recipe its very sus to only be possible to get from lockers

Reply:
The list got re-sorted, it's not being removed.